### ISSUE TITLE
Unable to get data from robot

### ISSUE THREAD

**Issue by roman2810** [koalazak/rest980#70]
Hi! I have issue with work of this add-on. I have configured it with the following parameters:

- PORT = 3000
- BLID
- PASSWORD
- ROBOT_IP
- FIRMWARE_VERSION = 2
- ENABLE_LOCAL = yes
- ENABLE_CLOUD = yes
- KEEP_ALIVE = yes
Add-on manages to start properly. But in add-on logs I can see only 2 recordings whenever I start any job from phone app:
![IMG_0572](https://user-images.githubusercontent.com/57016549/167854702-50d24695-3117-4afe-ae53-1e08a570682b.jpg)
If I try to do any GET request by myself, asking for example state I get error message.

BLID and PASSWORD were fetched, using roombapw Docker Image addon, so they should be correct.

Could you please advise what am I doing wrong and how to fix that?
Thanks in advance!

**Comment by mikesalz** [comment]
I spent hours trying to get this to work, as well.  But I finally got it working.  I think the missing piece is one additional variable in the dock er container - ROBOT_CIPHERS=TLS_AES_256_GCM_SHA384

**Comment by roman2810** [comment]
Well, I managed to make it work eventually. The thing was to disable any other client to the robot completely (phone app in my case did not count). 
For instance in my case I have official iRobot integration, installed on my Home Assistant instance. Disabling "Continuous" option for it (like it is described in the manual) did not work in my case.
![image](https://user-images.githubusercontent.com/57016549/172068056-0ea1cac9-b204-41ec-96ec-832897ab1991.png)
I had to disable this integration completely.
![image](https://user-images.githubusercontent.com/57016549/172068089-95485f03-ab05-4553-8f25-1d23a6a00748.png)
Only after that I managed to make this program connect to my robot. And I used it for both cases: launching it on my laptop and launching it as container add-on on my Home Assistant.
